src/common/metrics/hospitalizations.ts

```diff
@@ -0,0 +1,49 @@
+import { COLOR_MAP } from 'common/colors';
+import { Level, LevelInfoMap } from 'common/level';
+
+export const METRIC_NAME = 'ICU headroom used';
+
+const SHORT_DESCRIPTION_LOW = 'Can likely handle a new wave of COVID';
+const SHORT_DESCRIPTION_MEDIUM = 'At risk to a new wave of COVID';
+const SHORT_DESCRIPTION_HIGH = 'Unable to handle a new wave of COVID';
+const SHORT_DESCRIPTION_UNKNOWN = 'Insufficient data to assess';
+
+const LIMIT_LOW = 0.7;
+const LIMIT_MEDIUM = 0.95;
```

Comment:
I think you reverted to our old limits

These should be:
```
const LIMIT_LOW = 0.5;
const LIMIT_MEDIUM = 0.7;
```
?

Reply:
yessss yikes good catch